frontend/src/app/components/wp-query-select/wp-query-select-dropdown.component.ts

```diff
@@ -435,6 +437,11 @@ export class WorkPackageQuerySelectDropdownComponent implements OnInit, OnDestro
       params,
       opts
     );
+
+    // On mobile, close the main menu after selecting a view.
+    if (window.innerWidth < 680) {
```

Comment:
Can you move that into the toggleService? It will get lost here. Something like `this.toggleService.closeWhenOnMobile()` or something

Reply:
@oliverguenther crap, just merged it into `dev`. Yes, I'll move that.